tests/templates/components/test_radios_with_hero.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+
+def test_radios_with_hero_on_design_system_v2():
```

Comment:
we're already on 3.0.0

```suggestion
def test_radios_with_hero_on_design_system_v3():
```

Reply:
meh I already bumped this and it must've got lost, changes incoming